rust/operator-binary/src/backend/mod.rs

```diff
@@ -172,10 +178,13 @@ impl SecretVolumeSelector {
     /// Returns all addresses associated with a certain [`SecretScope`]
     fn scope_addresses<'a>(
         &'a self,
+        cluster_info: &KubernetesClusterInfo,
```

Comment:
Threading this kind of data around is what `PodInfo` exists for.

Reply:
```patch
diff --git a/rust/operator-binary/src/backend/cert_manager.rs b/rust/operator-binary/src/backend/cert_manager.rs
index 40a4e53..47cee77 100644
--- a/rust/operator-binary/src/backend/cert_manager.rs
+++ b/rust/operator-binary/src/backend/cert_manager.rs
@@ -10,7 +10,6 @@ use stackable_operator::{
     k8s_openapi::{api::core::v1::Secret, ByteString},
     kube::{api::ObjectMeta, runtime::reflector::ObjectRef},
     time::Duration,
-    utils::cluster_info::KubernetesClusterInfo,
 };
 
 use crate::{crd::CertManagerIssuer, external_crd, format::SecretData, utils::Unloggable};
@@ -84,7 +83,6 @@ impl SecretBackend for CertManager {
 
     async fn get_secret_data(
         &self,
-        cluster_info: &KubernetesClusterInfo,
         selector: &SecretVolumeSelector,
         pod_info: PodInfo,
     ) -> Result<SecretContents, Self::Error> {
@@ -97,7 +95,7 @@ impl SecretBackend for CertManager {
         let mut ip_addresses = Vec::new();
         for scope in &selector.scope {
             for address in selector
-                .scope_addresses(cluster_info, &pod_info, scope)
+                .scope_addresses(&pod_info, scope)
                 .context(ScopeAddressesSnafu { scope })?
             {
                 match address {
diff --git a/rust/operator-binary/src/backend/dynamic.rs b/rust/operator-binary/src/backend/dynamic.rs
index 72b09de..e37c891 100644
--- a/rust/operator-binary/src/backend/dynamic.rs
+++ b/rust/operator-binary/src/backend/dynamic.rs
@@ -7,9 +7,7 @@ use std::{
 
 use async_trait::async_trait;
 use snafu::{ResultExt, Snafu};
-use stackable_operator::{
-    kube::runtime::reflector::ObjectRef, utils::cluster_info::KubernetesClusterInfo,
-};
+use stackable_operator::kube::runtime::reflector::ObjectRef;
 
 use crate::{
     crd::{self, SecretClass},
@@ -60,12 +58,11 @@ impl<B: SecretBackend + Send + Sync> SecretBackend for DynamicAdapter<B> {
 
     async fn get_secret_data(
         &self,
-        cluster_info: &KubernetesClusterInfo,
         selector: &super::SecretVolumeSelector,
         pod_info: PodInfo,
     ) -> Result<super::SecretContents, Self::Error> {
         self.0
-            .get_secret_data(cluster_info, selector, pod_info)
+            .get_secret_data(selector, pod_info)
             .await
             .map_err(|err| DynError(Box::new(err)))
     }
diff --git a/rust/operator-binary/src/backend/k8s_search.rs b/rust/operator-binary/src/backend/k8s_search.rs
index f2dfa47..f0d9fe7 100644
--- a/rust/operator-binary/src/backend/k8s_search.rs
+++ b/rust/operator-binary/src/backend/k8s_search.rs
@@ -10,7 +10,6 @@ use stackable_operator::{
     },
     kube::api::ListParams,
     kvp::{LabelError, LabelSelectorExt, Labels},
-    utils::cluster_info::KubernetesClusterInfo,
 };
 
 use crate::{crd::SearchNamespace, format::SecretData, utils::Unloggable};
@@ -83,7 +82,6 @@ impl SecretBackend for K8sSearch {
 
     async fn get_secret_data(
         &self,
-        _cluster_info: &KubernetesClusterInfo,
         selector: &SecretVolumeSelector,
         pod_info: PodInfo,
     ) -> Result<SecretContents, Self::Error> {
diff --git a/rust/operator-binary/src/backend/kerberos_keytab.rs b/rust/operator-binary/src/backend/kerberos_keytab.rs
index f5f196f..684611b 100644
--- a/rust/operator-binary/src/backend/kerberos_keytab.rs
+++ b/rust/operator-binary/src/backend/kerberos_keytab.rs
@@ -97,6 +97,7 @@ pub struct KerberosKeytab {
     profile: KerberosProfile,
     admin_keytab: Unloggable<Vec<u8>>,
     admin_principal: KerberosPrincipal,
+    kubernetes_cluster_info: KubernetesClusterInfo,
 }
 
 impl KerberosKeytab {
@@ -127,6 +128,7 @@ impl KerberosKeytab {
             profile,
             admin_keytab: Unloggable(admin_keytab),
             admin_principal,
+            kubernetes_cluster_info: client.kubernetes_cluster_info.clone(),
         })
     }
 }
@@ -137,7 +139,6 @@ impl SecretBackend for KerberosKeytab {
 
     async fn get_secret_data(
         &self,
-        cluster_info: &KubernetesClusterInfo,
         selector: &super::SecretVolumeSelector,
         pod_info: super::pod_info::PodInfo,
     ) -> Result<super::SecretContents, Self::Error> {
@@ -150,6 +151,7 @@ impl SecretBackend for KerberosKeytab {
                 },
             admin_keytab,
             admin_principal,
+            kubernetes_cluster_info,
         } = self;
 
         let admin_server_clause = match admin {
@@ -203,11 +205,12 @@ cluster.local = {realm_name}
         let mut pod_principals: Vec<KerberosPrincipal> = Vec::new();
         for service_name in &selector.kerberos_service_names {
             for scope in &selector.scope {
-                for addr in selector
-                    .scope_addresses(cluster_info, &pod_info, scope)
-                    .context(ScopeAddressesSnafu {
-                        scope: scope.clone(),
-                    })?
+                for addr in
+                    selector
+                        .scope_addresses(&pod_info, scope)
+                        .context(ScopeAddressesSnafu {
+                            scope: scope.clone(),
+                        })?
                 {
                     if let Address::Dns(hostname) = addr {
                         pod_principals.push(
@@ -262,7 +265,7 @@ cluster.local = {realm_name}
                     },
                 },
             },
-            cluster_info,
+            kubernetes_cluster_info,
         )
         .await
         .context(ProvisionKeytabSnafu)?;
diff --git a/rust/operator-binary/src/backend/mod.rs b/rust/operator-binary/src/backend/mod.rs
index dbcadd5..7c2cdad 100644
--- a/rust/operator-binary/src/backend/mod.rs
+++ b/rust/operator-binary/src/backend/mod.rs
@@ -14,7 +14,6 @@ use snafu::{OptionExt, Snafu};
 use stackable_operator::{
     k8s_openapi::chrono::{DateTime, FixedOffset},
     time::Duration,
-    utils::cluster_info::KubernetesClusterInfo,
 };
 use std::{collections::HashSet, convert::Infallible, fmt::Debug};
 
@@ -178,12 +177,11 @@ impl SecretVolumeSelector {
     /// Returns all addresses associated with a certain [`SecretScope`]
     fn scope_addresses<'a>(
         &'a self,
-        cluster_info: &KubernetesClusterInfo,
         pod_info: &'a pod_info::PodInfo,
         scope: &scope::SecretScope,
     ) -> Result<Vec<Address>, ScopeAddressesError> {
         use scope_addresses_error::*;
-        let cluster_domain = &cluster_info.cluster_domain;
+        let cluster_domain = &pod_info.kubernetes_cluster_domain;
         let namespace = &self.namespace;
         Ok(match scope {
             scope::SecretScope::Node => {
@@ -272,7 +270,6 @@ pub trait SecretBackend: Debug + Send + Sync {
     /// Provision or load secret data from the source.
     async fn get_secret_data(
         &self,
-        cluster_info: &KubernetesClusterInfo,
         selector: &SecretVolumeSelector,
         pod_info: pod_info::PodInfo,
     ) -> Result<SecretContents, Self::Error>;
diff --git a/rust/operator-binary/src/backend/pod_info.rs b/rust/operator-binary/src/backend/pod_info.rs
index e1f65ce..2185974 100644
--- a/rust/operator-binary/src/backend/pod_info.rs
+++ b/rust/operator-binary/src/backend/pod_info.rs
@@ -8,7 +8,10 @@ use std::{
 use futures::{StreamExt, TryStreamExt};
 use snafu::{OptionExt, ResultExt, Snafu};
 use stackable_operator::{
-    commons::listener::{AddressType, Listener, ListenerClass, PodListeners, ServiceType},
+    commons::{
+        listener::{AddressType, Listener, ListenerClass, PodListeners, ServiceType},
+        networking::DomainName,
+    },
     k8s_openapi::api::core::v1::{Node, PersistentVolumeClaim, Pod},
     kube::runtime::reflector::ObjectRef,
 };
@@ -108,6 +111,7 @@ pub struct PodInfo {
     pub node_name: String,
     pub node_ips: Vec<IpAddr>,
     pub listener_addresses: HashMap<String, Vec<Address>>,
+    pub kubernetes_cluster_domain: DomainName,
     pub scheduling: SchedulingPodInfo,
 }
 
@@ -166,6 +170,7 @@ impl PodInfo {
                 })
                 .collect::<Result<_, _>>()?,
             listener_addresses,
+            kubernetes_cluster_domain: client.kubernetes_cluster_info.cluster_domain.clone(),
             scheduling,
         })
     }
diff --git a/rust/operator-binary/src/backend/tls/mod.rs b/rust/operator-binary/src/backend/tls/mod.rs
index a6ed8fb..4b5c06b 100644
--- a/rust/operator-binary/src/backend/tls/mod.rs
+++ b/rust/operator-binary/src/backend/tls/mod.rs
@@ -24,7 +24,6 @@ use snafu::{OptionExt, ResultExt, Snafu};
 use stackable_operator::{
     k8s_openapi::chrono::{self, FixedOffset, TimeZone},
     time::Duration,
-    utils::cluster_info::KubernetesClusterInfo,
 };
 use time::OffsetDateTime;
 
@@ -180,7 +179,6 @@ impl SecretBackend for TlsGenerate {
     /// Then add the ca certificate and return these files for provisioning to the volume.
     async fn get_secret_data(
         &self,
-        cluster_info: &KubernetesClusterInfo,
         selector: &super::SecretVolumeSelector,
         pod_info: PodInfo,
     ) -> Result<SecretContents, Self::Error> {
@@ -251,7 +249,7 @@ impl SecretBackend for TlsGenerate {
         for scope in &selector.scope {
             addresses.extend(
                 selector
-                    .scope_addresses(cluster_info, &pod_info, scope)
+                    .scope_addresses(&pod_info, scope)
                     .context(ScopeAddressesSnafu { scope })?,
             );
         }
diff --git a/rust/operator-binary/src/csi_server/node.rs b/rust/operator-binary/src/csi_server/node.rs
index 2ad8355..1aa8a75 100644
--- a/rust/operator-binary/src/csi_server/node.rs
+++ b/rust/operator-binary/src/csi_server/node.rs
@@ -375,7 +375,7 @@ impl Node for SecretProvisionerNode {
                 let pod_ref = ObjectRef::<Pod>::new(&selector.pod).within(&selector.namespace);
                 tracing::info!(pod = %pod_ref, ?selector, ?pod_info, ?backend, "issuing secret for Pod");
                 let data = backend
-                    .get_secret_data(&self.client.kubernetes_cluster_info, &selector, pod_info)
+                    .get_secret_data(&selector, pod_info)
                     .await
                     .context(publish_error::BackendGetSecretDataSnafu)?;
                 self.tag_pod(&self.client, &request.volume_id, &selector, &data)
```

Reply:
This kind of highlights the issue around having to special-case-throw it into krb5-pt though...

Reply:
Divesting krb5-pt from op-rs shouldn't be too bad, either:

```patch
diff --git a/Cargo.lock b/Cargo.lock
index 6fd23dc..fb5d6c4 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -2707,7 +2707,6 @@ name = "stackable-krb5-provision-keytab"
 version = "0.0.0-dev"
 dependencies = [
  "byteorder",
- "clap",
  "futures 0.3.31",
  "krb5",
  "ldap3",
diff --git a/rust/krb5-provision-keytab/Cargo.toml b/rust/krb5-provision-keytab/Cargo.toml
index 66ab0c2..b90bca3 100644
--- a/rust/krb5-provision-keytab/Cargo.toml
+++ b/rust/krb5-provision-keytab/Cargo.toml
@@ -13,7 +13,6 @@ krb5 = { path = "../krb5" }
 stackable-secret-operator-crd-utils = { path = "../crd-utils" }
 
 byteorder.workspace = true
-clap.workspace = true
 futures.workspace = true
 ldap3.workspace = true
 native-tls.workspace = true
diff --git a/rust/krb5-provision-keytab/src/active_directory.rs b/rust/krb5-provision-keytab/src/active_directory.rs
index 40cec94..63f8356 100644
--- a/rust/krb5-provision-keytab/src/active_directory.rs
+++ b/rust/krb5-provision-keytab/src/active_directory.rs
@@ -11,8 +11,8 @@ use rand::{seq::SliceRandom, thread_rng, CryptoRng};
 use snafu::{OptionExt, ResultExt, Snafu};
 use stackable_krb5_provision_keytab::ActiveDirectorySamAccountNameRules;
 use stackable_operator::{
-    k8s_openapi::api::core::v1::Secret, kube::runtime::reflector::ObjectRef,
-    utils::cluster_info::KubernetesClusterInfoOpts,
+    k8s_openapi::api::core::v1::Secret,
+    kube::{self, runtime::reflector::ObjectRef},
 };
 use stackable_secret_operator_crd_utils::SecretReference;
 
@@ -22,7 +22,7 @@ use crate::credential_cache::{self, CredentialCache};
 pub enum Error {
     #[snafu(display("failed to retrieve LDAP TLS CA {ca_ref}"))]
     GetLdapTlsCa {
-        source: stackable_operator::client::Error,
+        source: kube::Error,
         ca_ref: ObjectRef<Secret>,
     },
 
@@ -45,9 +45,7 @@ pub enum Error {
     LdapAuthn { source: ldap3::LdapError },
 
     #[snafu(display("failed to init Kubernetes client"))]
-    KubeInit {
-        source: stackable_operator::client::Error,
-    },
+    KubeInit { source: kube::Error },
 
     #[snafu(display("failed to unparse Kerberos principal"))]
     UnparsePrincipal { source: krb5::Error },
@@ -95,7 +93,6 @@ pub struct AdAdmin<'a> {
 
 impl<'a> AdAdmin<'a> {
     pub async fn connect(
-        cluster_info_opts: &KubernetesClusterInfoOpts,
         ldap_server: &str,
         krb: &'a KrbContext,
         ldap_tls_ca_secret: SecretReference,
@@ -104,9 +101,7 @@ impl<'a> AdAdmin<'a> {
         schema_distinguished_name: String,
         generate_sam_account_name: Option<ActiveDirectorySamAccountNameRules>,
     ) -> Result<AdAdmin<'a>> {
-        let kube = stackable_operator::client::initialize_operator(None, cluster_info_opts)
-            .await
-            .context(KubeInitSnafu)?;
+        let kube = kube::Client::try_default().await.context(KubeInitSnafu)?;
         let ldap_tls = native_tls::TlsConnector::builder()
             .disable_built_in_roots(true)
             .add_root_certificate(get_ldap_ca_certificate(&kube, ldap_tls_ca_secret).await?)
@@ -184,11 +179,12 @@ impl<'a> AdAdmin<'a> {
 }
 
 async fn get_ldap_ca_certificate(
-    kube: &stackable_operator::client::Client,
+    kube: &kube::Client,
     ca_secret_ref: SecretReference,
 ) -> Result<native_tls::Certificate> {
-    let ca_secret = kube
-        .get::<Secret>(&ca_secret_ref.name, &ca_secret_ref.namespace)
+    let secrets = kube::Api::<Secret>::namespaced(kube.clone(), &ca_secret_ref.namespace);
+    let ca_secret = secrets
+        .get(&ca_secret_ref.name)
         .await
         .context(GetLdapTlsCaSnafu {
             ca_ref: ca_secret_ref,
diff --git a/rust/krb5-provision-keytab/src/credential_cache.rs b/rust/krb5-provision-keytab/src/credential_cache.rs
index c70d77a..5b87e8a 100644
--- a/rust/krb5-provision-keytab/src/credential_cache.rs
+++ b/rust/krb5-provision-keytab/src/credential_cache.rs
@@ -2,7 +2,11 @@ use futures::{TryFuture, TryFutureExt};
 use snafu::{OptionExt, ResultExt, Snafu};
 use stackable_operator::{
     k8s_openapi::{api::core::v1::Secret, ByteString},
-    kube::runtime::reflector::ObjectRef,
+    kube::{
+        self,
+        api::{Patch, PatchParams},
+        runtime::reflector::ObjectRef,
+    },
 };
 use stackable_secret_operator_crd_utils::SecretReference;
 
@@ -10,13 +14,13 @@ use stackable_secret_operator_crd_utils::SecretReference;
 pub enum Error {
     #[snafu(display("failed to load initial cache from {cache_ref}"))]
     GetInitialCache {
-        source: stackable_operator::client::Error,
+        source: kube::Error,
         cache_ref: ObjectRef<Secret>,
     },
 
     #[snafu(display("failed to save credential {key} to {cache_ref}"))]
     SaveToCache {
-        source: stackable_operator::client::Error,
+        source: kube::Error,
         key: String,
         cache_ref: ObjectRef<Secret>,
     },
@@ -31,7 +35,7 @@ type Result<T, E = Error> = std::result::Result<T, E>;
 
 pub struct CredentialCache {
     name: &'static str,
-    kube: stackable_operator::client::Client,
+    secrets: kube::Api<Secret>,
     cache_ref: SecretReference,
     current_state: Secret,
 }
@@ -39,19 +43,20 @@ impl CredentialCache {
     #[tracing::instrument(skip(kube))]
     pub async fn new(
         name: &'static str,
-        kube: stackable_operator::client::Client,
+        kube: kube::Client,
         cache_ref: SecretReference,
     ) -> Result<Self> {
+        let secrets = kube::Api::<Secret>::namespaced(kube, &cache_ref.namespace);
         Ok(Self {
             name,
-            current_state: kube
-                .get::<Secret>(&cache_ref.name, &cache_ref.namespace)
+            current_state: secrets
+                .get(&cache_ref.name)
                 .await
                 .context(GetInitialCacheSnafu {
                     cache_ref: &cache_ref,
                 })?,
             cache_ref,
-            kube,
+            secrets,
         })
     }
 
@@ -95,13 +100,14 @@ impl CredentialCache {
                 Ok(value) => {
                     tracing::info!("generated credential successfully, saving...");
                     self.current_state = self
-                        .kube
-                        .merge_patch(
-                            &self.current_state,
-                            &Secret {
+                        .secrets
+                        .patch(
+                            &self.cache_ref.name,
+                            &PatchParams::default(),
+                            &Patch::Merge(Secret {
                                 data: Some([(key.to_string(), ByteString(value))].into()),
                                 ..Secret::default()
-                            },
+                            }),
                         )
                         .await
                         .context(SaveToCacheSnafu {
diff --git a/rust/krb5-provision-keytab/src/lib.rs b/rust/krb5-provision-keytab/src/lib.rs
index 63a5a0b..41181b8 100644
--- a/rust/krb5-provision-keytab/src/lib.rs
+++ b/rust/krb5-provision-keytab/src/lib.rs
@@ -7,7 +7,6 @@ use std::{
 
 use serde::{Deserialize, Serialize};
 use snafu::{ResultExt, Snafu};
-use stackable_operator::utils::cluster_info::KubernetesClusterInfo;
 use stackable_secret_operator_crd_utils::SecretReference;
 use tokio::{io::AsyncWriteExt, process::Command};
 
@@ -68,16 +67,10 @@ pub enum Error {
 /// Provisions a Kerberos Keytab based on the [`Request`].
 ///
 /// This function assumes that the binary produced by this crate is on the `$PATH`, and will fail otherwise.
-pub async fn provision_keytab(
-    krb5_config_path: &Path,
-    req: &Request,
-    cluster_info: &KubernetesClusterInfo,
-) -> Result<Response, Error> {
+pub async fn provision_keytab(krb5_config_path: &Path, req: &Request) -> Result<Response, Error> {
     let req_str = serde_json::to_vec(&req).context(SerializeRequestSnafu)?;
 
-    let args = vec!["--kubernetes-cluster-domain", &cluster_info.cluster_domain];
     let mut child = Command::new("stackable-krb5-provision-keytab")
-        .args(args)
         .kill_on_drop(true)
         .env("KRB5_CONFIG", krb5_config_path)
         // ldap3 uses the default client keytab to authenticate to the LDAP server
diff --git a/rust/krb5-provision-keytab/src/main.rs b/rust/krb5-provision-keytab/src/main.rs
index 3020979..137762c 100644
--- a/rust/krb5-provision-keytab/src/main.rs
+++ b/rust/krb5-provision-keytab/src/main.rs
@@ -4,11 +4,9 @@ use std::{
     io::{stdin, BufReader},
 };
 
-use clap::Parser;
 use krb5::{Keyblock, Keytab};
 use snafu::{ResultExt, Snafu};
 use stackable_krb5_provision_keytab::{AdminBackend, Request, Response};
-use stackable_operator::utils::cluster_info::KubernetesClusterInfoOpts;
 use tracing::info;
 
 mod active_directory;
@@ -69,20 +67,12 @@ enum Error {
     RemoveDummyFromKeytab { source: krb5::Error },
 }
 
-#[derive(clap::Parser)]
-pub struct Args {
-    #[command(flatten)]
-    pub cluster_info_opts: KubernetesClusterInfoOpts,
-}
-
 enum AdminConnection<'a> {
     Mit(mit::MitAdmin<'a>),
     ActiveDirectory(active_directory::AdAdmin<'a>),
 }
 
 async fn run() -> Result<Response, Error> {
-    let args = Args::parse();
-
     let req = serde_json::from_reader::<_, Request>(BufReader::new(stdin().lock()))
         .context(DeserializeRequestSnafu)?;
     info!("initing context");
@@ -107,7 +97,6 @@ async fn run() -> Result<Response, Error> {
             generate_sam_account_name,
         } => AdminConnection::ActiveDirectory(
             active_directory::AdAdmin::connect(
-                &args.cluster_info_opts,
                 &ldap_server,
                 &krb,
                 ldap_tls_ca_secret,
diff --git a/rust/operator-binary/src/backend/kerberos_keytab.rs b/rust/operator-binary/src/backend/kerberos_keytab.rs
index 684611b..9bce7f3 100644
--- a/rust/operator-binary/src/backend/kerberos_keytab.rs
+++ b/rust/operator-binary/src/backend/kerberos_keytab.rs
@@ -9,7 +9,6 @@ use stackable_operator::{
     commons::networking::{HostName, KerberosRealmName},
     k8s_openapi::api::core::v1::Secret,
     kube::runtime::reflector::ObjectRef,
-    utils::cluster_info::KubernetesClusterInfo,
 };
 use stackable_secret_operator_crd_utils::SecretReference;
 use tempfile::tempdir;
@@ -97,7 +96,6 @@ pub struct KerberosKeytab {
     profile: KerberosProfile,
     admin_keytab: Unloggable<Vec<u8>>,
     admin_principal: KerberosPrincipal,
-    kubernetes_cluster_info: KubernetesClusterInfo,
 }
 
 impl KerberosKeytab {
@@ -128,7 +126,6 @@ impl KerberosKeytab {
             profile,
             admin_keytab: Unloggable(admin_keytab),
             admin_principal,
-            kubernetes_cluster_info: client.kubernetes_cluster_info.clone(),
         })
     }
 }
@@ -151,7 +148,6 @@ impl SecretBackend for KerberosKeytab {
                 },
             admin_keytab,
             admin_principal,
-            kubernetes_cluster_info,
         } = self;
 
         let admin_server_clause = match admin {
@@ -265,7 +261,6 @@ cluster.local = {realm_name}
                     },
                 },
             },
-            kubernetes_cluster_info,
         )
         .await
         .context(ProvisionKeytabSnafu)?;
```

Reply:
Applied the first suggestion in a8903fc, thanks!

Reply:
Regarding the migration to plain kube, I was at the same point (slight differences with variables but anyway).
The point where I stopped is `&PatchParams::default()`, where op-rs is setting the field_manager.
And I had the feeling that we are loosing some quality-of-life features from op-rs, but I guess the implementation is rather oversee-able, so these are not urgently needed.

Anyway, I would commit your suggestion and add the field_manager to the PatchParams

Reply:
We don't set a field manager already (we pass `None` to `initialize_operator`), but I wouldn't be against adding one.

Reply:
I added the field_manager `secrets.stackable.tech/krb5-provision-keytab` in ab36e7e, hope that works.